config/locales/es.yml

```diff
@@ -784,9 +784,9 @@ es:
         allows_greeters: Permite saludadores
         excludes: 'Excluye estados: Aceptado, No Archivar, en En Pausa'
       index:
-        add_coalition: Add new coalition
-        add_organization: Add new organization
-        title: Organization List
+        add_coalition: Agregar una nueva coalición
+        add_organization: Agregar una nueva coalición
```

Comment:
This seems incorrect to me, but I've brought it up on Slack now, so I don't mind merging this anyway.